root/release/edit/recordings.tt

```diff
@@ -169,9 +169,20 @@
 
       <table data-bind="with: target">
         <tbody>
+          <tr>
+            <td>
+              <input name="recording-selection" value="" type="radio" id="add-new-recording" data-bind="checked: hasNewRecording, checkedValue: true" />
+            </td>
+            <td colspan="3">
+              <label for="add-new-recording">
+                [% l('Add a new recording') %]
+              </label>
+            </td>
+          </tr>
+
           <!-- ko if: loadingSuggestedRecordings -->
           <tr>
-            <td colspan="4" style="padding-bottom: 1em; vertical-align: top;" class="loading-message">
+            <td colspan="4" style="background-position: left 1em; padding-bottom: 1em; padding-top: 1em; vertical-align: top;" class="loading-message">
```

Comment:
I'm not super-happy with this, but I didn't see a better way to handle the loading message being smooshed up against the "Add a new recording" row.

I don't want to set `padding-bottom` on "Add a new recording" since the extra spacing remains after this message disappears. When I set `padding-top` here, I also need to update `background-position` so that the spinner added by the `loading-message` class is aligned with the text.

Other possibilities might be changing `display` so I can set `margin-top`, or adding an empty row within the `loadingSuggestedRecordings` condition just to add spacing. Let me know if either of those sound better.